subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -1462,6 +1494,26 @@ static bool pdu_validate_cte_resp(struct pdu_data *pdu)
 }
 #endif /* CONFIG_BT_CTLR_DF_CONN_CTE_RSP */
 
+#if defined(CONFIG_BT_CTLR_SCA_UPDATE)
+static bool pdu_validate_clock_accuracy_req(struct pdu_data *pdu)
+{
+	if (pdu->len != sizeof(pdu->llctrl.sca_req) + 1) {
+		return false;
+	}
+
+	return true;
+}
+#endif /* CONFIG_BT_CTLR_DATA_LENGTH */
+
+static bool pdu_validate_clock_accuracy_rsp(struct pdu_data *pdu)
+{
+	if (pdu->len != sizeof(pdu->llctrl.sca_rsp) + 1) {
```

Comment:
```suggestion
	if (pdu->len != sizeof(pdu->llctrl.sca_rsp) + 1U) {
```

Reply:
I suggest to accept this, as this repeats a pattern for all validation funcs.
I will follow-up with a PR changing this, using a macro instead.